### PR TITLE
Devops in a box

### PR DESCRIPTION
**Pull request by shivani170**
Updated DevOps in a box

Fixes https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/503/
Screenshot 2022-09-13 at 10 46 37 PM

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -115,8 +115,9 @@
     <header class="">
         <div class="padding_ticker" style="width: 100%; background-color: #3e3ead; text-align: center; position: fixed; top: 0px; z-index: 1000; align-items: center;">
             <div>
-                <a onclick='viewOnDevopsInABoxBannerGAEvent("Banner")' href="./support.html" class="support-program__k8s" style="position: relative; padding: 12px 0px 12px 0px; background-color: transparent; display: block;">
-                    <span>🔥 DevOps in a Box 🎁 </span>
+                <a onclick='viewOnDevopsInABoxBannerGAEvent("Banner")' href="./support.html" class="support-program__k8s" style="position: relative; padding: 12px 0px 12px 0px; background: linear-gradient(90deg, #8314A2 0%, #3E3EAD 100%);
+                ; display: block;">
```
Comment (sohel-rp) [inline public/index.html:119]: remove extra ;

File: public/support.html [hunk 2]
```diff
@@ -94,29 +94,32 @@
     <link rel="stylesheet" href="css/header.css" />
     <link rel="stylesheet" href="css/styles.css?time=16258199860193" />
     <link rel="stylesheet" href=" https://cdn.devtron.ai/css/animations.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (sohel-rp) [inline public/support.html:98]: We have already preconnected these 2 at top

File: public/support.html [hunk 3]
```diff
@@ -94,29 +94,32 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Barlow+Condensed:wght@300;600&display=swap" rel="stylesheet">
```
Comment (sohel-rp) [inline public/support.html:99]: Try to get this font family along with `Noto+Serif` from above

File: public/index.html [hunk 4]
```diff
@@ -129,12 +130,17 @@
               </div>
               <a href="index.html" aria-current="page" class="brand w-nav-brand w--current"><img src="images/Logo.svg" loading="lazy" alt=""></a>
               <nav role="navigation" class="nav-menu w-nav-menu">
-                <a href="./support.html" class="nav-link white w-nav-link" onclick='viewOnDevopsInABoxGAEvent("Nav")'>#DevOpsInABox 🎁</a>
```
Comment (vivek-devtron) [inline public/index.html:132]: is removal of box icon expected?

File: public/index.html [hunk 5]
```diff
@@ -129,12 +130,17 @@
...
+                <a href="./support.html" class="nav-link white w-nav-link" onclick='viewOnDevopsInABoxGAEvent("Nav")'>#DevOpsInABox</a>
                 <a href="https://docs.devtron.ai/" class="nav-link white w-nav-link" target="_blank">Docs</a>
                 <a href="team.html" aria-current="page" class="nav-link white w-nav-link">Team</a>
                 <a href="careers.html" class="nav-link white w-nav-link">Careers</a>
                 <a href="https://devtron.ai/blog/" class="nav-link white w-nav-link" target="_blank">Blog</a>
-                <a href=" https://preview.devtron.ai/dashboard/" class="demo-button tab-view" target="_blank" onclick="demoHeader('homepage')">Demo <img src="images/arrow-right.svg"></a>
```
Comment (vivek-devtron) [inline public/index.html:137]: is removal of onclick="demoHeader('homepage')" is expected?

File: public/support.html [hunk 6]
```diff
@@ -695,54 +704,111 @@ <h2 class="mb-0">
...
+                src="./images/trusted-by/logo-appyhigh.png"
                 class="trusted-by__item"
               />
             </span>
           </div>
         </div>
       </section>
 
-      <section class="bg-white">
+      <!-- <section class="bg-white">
```
Comment (vivek-devtron) [inline public/support.html:802]: we can remove commented code

File: public/index.html (hunk 5 above)
Comment (shivani170) [reply public/index.html:137]: Replaced with Try Devtron

Comment (shivani170) [reply public/index.html:137]: Added

File: public/index.html (hunk 4 above)
Comment (shivani170) [reply public/index.html:132]: Yes

File: public/support.html (hunk 3 above)
Comment (shivani170) [reply public/support.html]: Are they both same?

File: public/js/main.js [hunk 7]
```diff
@@ -353,6 +360,11 @@ function tryDevtronHero() {
   });
 }
 
+function tryDevtronVieoSection() {
```
Comment (vivek-devtron) [inline public/js/main.js:363]: tryDevtronVieoSection to tryDevtronVideoSection

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**